src/cozy-ui.d.ts

```diff
@@ -77,6 +77,7 @@ declare module 'cozy-ui/transpiled/react/TextField' {
 }
 
 declare module 'cozy-ui/transpiled/react/deprecated/Button'
```

Comment:
This line can be remove now 🤔

Reply:
non ils sont encore utilisés dans 2 composants compliqués à tester donc j'ai laissé comme tel. Les boutons dépréciés sont quand même compatible dark mode c'est pas trop un souci.